sys/parsoid.js

```diff
@@ -621,9 +621,7 @@ PSP.transformRevision = function(hyper, req, from, to) {
             original: original
         };
         if (from === 'sections') {
-            body2.html = {
-                body: replaceSections(original, parseSections(req))
-            };
+            body2.html = replaceSections(original, parseSections(req));
```

Comment:
@subbuss would sympathize with your sentiment.  I'm working on getting better at that.